### DISCUSSION TITLE
April 28, 2023 (previously planned to Mar 10) – Lab updates, deprecation, and Content Bug fixes

### DISCUSSION THREAD

**Discussion by LuizMacedo** [MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions#394] [Announcements]
Update: **MINOR**
Type of change: **LABS/SLIDES/CONTENT**

**LABS**
Continuing with AZ-400 Lab updates, we have new labs replacing Azure Demo Generator by eShopOnWeb - https://github.com/MicrosoftLearning/eShopOnWeb.
Also, we are deprecating labs, and one due to the DevOps Starter retirement planned for March 31, 2023 - https://learn.microsoft.com/azure/devops-project/retirement-and-migration.

**Labs refreshed:**

- Enabling Continuous Integration with Azure Pipelines.

**Labs deprecated:**

- Creating a Release Dashboard.
- Integrating External Source Control with Azure Pipelines.
- Integration between Azure DevOps and Teams.

**SLIDES**

- Minor typos and word capitalizations corrected.

**CONTENT**

- Overall content bug fixes. Types, word capitalizations, grammar, etc.

**Comment by Ellywati** [comment]
It is still in the lab instruction and cause confusion in the class, I have to announced that it is retired and many students are not happy that the instruction is still there while the DevOps has retired. Could this please be removed immediately as DevOps has retired on 31 March 2023? Thank you https://azure.microsoft.com/en-us/updates/retirement-notice-devops-starter-is-being-retired/

**Comment by unaihuete93** [comment]
@lauraAlvEl @petender @fimdim when are we retiring the mentioned labs?

**Comment by LuizMacedo** [comment]
@Ellywati, I'm sorry about the confusion it caused. We needed to postpone the update, but I'll update the change log with the new date and remove the deprecated labs as soon as possible.

**Comment by LuizMacedo** [comment]
I'm working on releasing the lab deprecation on Friday, 04/28. I'm sorry about any issues the delay caused.
